Assets/ZSSRichTextEditor.js

```diff
@@ -694,8 +694,7 @@ ZSSEditor.insertImage = function(url, alt) {
  *                                      does not check for that.  It would be a mistake.
  */
 ZSSEditor.insertLocalImage = function(imageNodeIdentifier, localImageUrl) {
-    // this hiddenChar helps with editing the content around images: http://stackoverflow.com/questions/18985261/cursor-in-wrong-place-in-contenteditable
-    var hiddenChar = '\ufeff';
+    var space = '&nbsp';
```

Comment:
I was going to mention that you forgot the semi-colon, but it looks like it is auto-inserted for ya. :-)